content/en/docs/reference/kube-scheduler/policies.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Kube-scheduler Policies
```

Comment:
@alculquicondor , Just checking on the terminology.
kube-scheduler is the name of the executable. How would you refer to the policies of the Kubernetes Scheduler in a generic way?

Reply:
This is outdated.